cibuildwheel/util.py

```diff
@@ -81,3 +81,31 @@ def download(url, dest):
             file.write(response.read())
     finally:
         response.close()
+
+
+class DependencyConstraints(object):
+    def __init__(self, base_file_path):
```

Comment:
In this PR there are different constraints for python 2.7 and 3.x. But if user would like to use custom constrains then need to run cibuildwheel twice. It is intentional?

Reply:
The same version-specific pattern applies for user-supplied constraints files too - this is documented in `docs/options.md`

Reply:
ok, my bad